content/docs/apps/production-ready.md

```diff
@@ -49,7 +49,7 @@ When running an application for development or testing, it is best to have a sep
 You want to receive alerts about application errors, downtime, and throughput issues.
 
 #### How
-* New Relic provides extensive application performance and availability monitoring with "Insights". You can set up and receive alerts on a variety of metrics.
+* There are many external services that provide alerting. For example, New Relic provides application performance and availability monitoring with "Insights". You can set up and receive alerts on a variety of metrics.
```

Comment:
FWIW, Insights is a big package containing several monitoring and analytics. The availability monitoring is specifically called "Synthetics" within Insights. I would consider the performance monitoring a slightly more advanced topic, so we could encourage users to start with Synthetics, and then dig into setting up more advanced monitoring.